src/app/pages/home/home.component.html

```diff
@@ -1,4 +1,8 @@
-<button (click)="addTask()">Add</button>
-<h1>Tusks</h1>
-<app-search-bar></app-search-bar>
-<app-tasks-list></app-tasks-list>
+<main>
+    <div id="header">
```

Comment:
In `desktop` and `tablet` resolutions, I would like to see the search bar beside the button, and both items should be in a smaller size